.binder/environment.yml

```diff
@@ -54,9 +54,10 @@ dependencies:
   - python >=3.10,<3.11
   # build
   - doit >=0.34,<1
-  - flit >=3.7.1,<4
+  - hatch >=1.6.3,<2
```

Comment:
this seems like a larger-than-needed change. no hate on `hatch`, but `flit` can ship shared data just fine.

Reply:
See https://github.com/jupyterlite/jupyterlite/pull/1013#discussion_r1141895631